### ISSUE TITLE
Unable to login due to missing directory for file keyring

### ISSUE THREAD

**Issue by flutcon** [kraudcloud/cli#2]
I downloaded and extracted the cli for MACOS darwin amd64. Than trying to login I get:

```
./kra login <my-token>

FATA[0000] No directory provided for file keyring
```
Do I need to configure something?

**Comment by aep** [comment]
sorry, for some reason i missed the notification for this bug.

A customer had the same issue today. Will look into it.

**Comment by astorozhevsky** [comment]
+1

**Comment by BertKiv** [comment]
+1

**Comment by karitham** [comment]
Alright, I pushed a temporary release you can find here https://github.com/kraudcloud/cli/releases/tag/v1.1.4 

It will ask for a password every time you use the cli, so the UX is quite honestly terrible.

I'm planning to fix this to use the native darwin keyring, but hitting classic C issues.

I'm keeping this open until it's properly fixed, since such a bad UX is unacceptable. Hopefully it's still usable

**Comment by karitham** [comment]
Fixed by 127256e8375126142fee5901805fafd8d59161f6.

The release action is building a cli which you can use without re-typing your password every time on darwin.
